wagtail/images/views/images.py

```diff
@@ -296,15 +296,26 @@ def add(request):
             image.file.seek(0)
             image._set_file_hash(image.file.read())
             image.file.seek(0)
+            is_duplicate = (
+                ImageModel.objects.filter(file_hash=image.file_hash).exists()
+            )
 
             form.save()
 
             # Reindex the image to make sure all tags are indexed
             search_index.insert_or_update_object(image)
 
-            messages.success(request, _("Image '{0}' added.").format(image.title), buttons=[
+            message = _("Image '{0}' added.").format(image.title)
+            buttons = [
                 messages.button(reverse('wagtailimages:edit', args=(image.id,)), _('Edit'))
-            ])
+            ]
+            messages_handler = messages.success
+            if is_duplicate:
+                message += _(" However it seems to be a duplicate.")
```

Comment:
Generally, try to lean away from concatenating translatable strings. Particularly since it has to start with a space, this could be confusing to translators, who might not get the associated context! It may be worth just repeating the start of the phrase, so a translator can adjust the whole "image added, but might be a duplicate" sentiment